api/History.json

```diff
@@ -497,10 +497,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "4"
```

Comment:
Here's Firefox 4 again, but http://mdn-bcd-collector.appspot.com/tests/api/History/state is so simple that I don't think it can be wrong. Confirmed it returns false in Firefox 3 too.